dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnector.java

```diff
@@ -345,6 +353,14 @@ private void dumpTable(
                 jsonSerializer.serialize(primaryKeys, writer);
                 writer.write(",\"foreignKeys\":");
                 jsonSerializer.serialize(foreignKeys, writer);
+                writer.write(",\"uniqueConstraints\":");
+                jsonSerializer.serialize(uniqueConstraints, writer);
+                writer.write(",\"nonNullConstraints\":");
+                jsonSerializer.serialize(nonNullConstraints, writer);
+                writer.write(",\"defaultConstraints\":");
+                jsonSerializer.serialize(defaultConstraints, writer);
+                writer.write(",\"checkConstraints\":");
+                jsonSerializer.serialize(checkConstraints, writer);
```

Comment:
Should we expect more items to be added here in the future?

Reply:
At the moment one more item is planned to be added - columnStats.

Reply:
I've changed it to a loop. Thanks!